### PR TITLE
enqueue imported RDF .nt elements for GPW

### PR DESCRIPTION
**Pull request by dsingley**
- [x] @sfeng88 @rygim @jharwig 
- [x] @srfarley @joeferner @mwizeman 
- [x] @EvanOxfeld @joeybrk372

### CODE REVIEW

File: common/rdf/src/main/java/org/visallo/common/rdf/RdfTripleImportHelper.java [hunk 1]
```diff
@@ -115,9 +144,17 @@ private void importRdfTriple(
 
             ++lineNum;
         }
+
+        if (workQueueRepository == null) {
```
Comment (joeferner) [inline common/rdf/src/main/java/org/visallo/common/rdf/RdfTripleImportHelper.java]: When would this happen?

File: common/rdf/src/main/java/org/visallo/common/rdf/RdfTripleImportHelper.java [hunk 2]
```diff
@@ -69,9 +69,35 @@ public RdfTripleImportHelper(
         this.visibilityTranslator = visibilityTranslator;
     }
 
+    @Inject
+    public void setWorkQueueRepository(WorkQueueRepository workQueueRepository) {
```
Comment (joeferner) [inline common/rdf/src/main/java/org/visallo/common/rdf/RdfTripleImportHelper.java]: Why isn't this part of the constructor?

File: common/rdf/src/main/java/org/visallo/common/rdf/RdfTripleImportHelper.java [hunk 3]
```diff
@@ -69,9 +69,35 @@ public RdfTripleImportHelper(
...
+        this.workQueueRepository = workQueueRepository;
+    }
+
+    @Deprecated
+    public void importRdfTriple(
```
Comment (joeferner) [inline common/rdf/src/main/java/org/visallo/common/rdf/RdfTripleImportHelper.java]: Who uses this? Can it be removed?

**Comment by joeferner** [comment]
Should this be optional?

 If 5 lines contribute to a vertex should we somehow wait to queue until they all go in? 

Graph flush needs to be called before queuing which is going to effect performance.

File: common/rdf/src/main/java/org/visallo/common/rdf/RdfTripleImportHelper.java (hunk 1 above)
Comment (srfarley) [inline common/rdf/src/main/java/org/visallo/common/rdf/RdfTripleImportHelper.java]: :+1: I think there should always be a queue present.

**Comment by joeferner** [comment]
LGTM after the changes

**Pull request merged**